docs/src/pages/en/jupyter-widgets-the-good-parts.mdx

```diff
@@ -415,7 +415,7 @@ return value is sent to the front end.
 
 A more complex serialization example might be to serialize a `pandas.DataFrame`
 to the [Apache Arrow](https://arrow.apache.org/#) format or a `numpy` array to
-its underlying bytes. Several [community examples](./community) should serve as
+its underlying bytes. Several [community examples](/community) should serve as
```

Comment:
```suggestion
its underlying bytes. Several [community examples](/en/community) should serve as
```

This is vestigial from the template I used, but every path is prefixed with the language (english)